packages/calling-stateful-client/src/DeviceManagerDeclarative.ts

```diff
@@ -16,6 +19,16 @@ export interface StatefulDeviceManager extends DeviceManager {
    * {@link StatefulDeviceManager}. See also {@link DeviceManagerState.selectedCamera}.
    */
   selectCamera: (VideoDeviceInfo) => void;
+
```

Comment:
I think this is neat solution. 
I do not quite understand why this is not most ideal solution according to your pr description. 
I'll approve as it seems okay to me, but I may rereview depending on better understanding